vignettes/datatable-importing.Rmd

```diff
@@ -198,3 +198,83 @@ Some of internally used C routines are now exported on C level thus can be used
 ## Importing from non-R Applications {non-r-API}
```

Comment:
I see some words being capitalized in the headings apart from the first word or proper nouns ('Applications' here, and more below), avoid that